tool/tbot/config/configtemplate.go

```diff
@@ -74,7 +73,7 @@ func (c *TemplateConfig) UnmarshalYAML(node *yaml.Node) error {
 		switch simpleTemplate {
 		case TemplateSSHClientName:
 			c.SSHClient = &TemplateSSHClient{}
-			fmt.Println("no params, using defaults")
+			log.Println("no params, using defaults")
```

Comment:
Whoops, I definitely should've removed that log entirely.

Reply:
I just removed it here then